code/__DEFINES/_flags.dm

```diff
@@ -155,6 +155,7 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define PASSITEM (1<<12)
 /// Do not intercept click attempts during Adjacent() checks. See [turf/proc/ClickCross]. **ONLY MEANINGFUL ON pass_flags_self!**
 #define LETPASSCLICKS (1<<13)
+#define PASSWINDOW (1<<14)
```

Comment:
doc

Reply:
the rest of these aren't docced

Reply:
other people being weak is not an excuse to nto do it.

Reply:
Fair